db/migrate/20150417194225_add_description_to_application_types.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToApplicationTypes < ActiveRecord::Migration
+  def change
+  	add_column :application_types, :description, :string, null: false, default: ""
```

Comment:
use text instead of string. We may get postgreSQL problems on the string 255 char limit.